src/Simplex/Messaging/Protocol.hs

```diff
@@ -1044,13 +1056,18 @@ class ProtocolMsgTag (Tag msg) => ProtocolEncoding err msg | msg -> err where
 instance PartyI p => ProtocolEncoding ErrorType (Command p) where
   type Tag (Command p) = CommandTag p
   encodeProtocol v = \case
-    NEW rKey dhKey auth_ -> case auth_ of
-      Just auth
-        | v >= 5 -> new <> e ('A', auth)
-        | otherwise -> new
-      _ -> new
+    NEW rKey dhKey subMode auth_
+      | v >= 6 -> new <> subModeEnc <> authEnc
+      | v == 5 -> new <> authEnc
+      | otherwise -> new
       where
         new = e (NEW_, ' ', rKey, dhKey)
+        subModeEnc = case subMode of
+          SMOnlyCreate -> e 'C'
+          SMSubscribe -> mempty
+        authEnc = case auth_ of
+          Just auth -> e ('A', auth)
+          Nothing -> mempty
```

Comment:
serialisation changed from A always present to sometimes present, so it's changed in v5.